.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
```suggestion
**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

Or

Example to reproduce issue with
``` text
example here
```
_Please make sure to include the rule(s) that cause the issue if applicable_

```

Reply:
It seems the ending three backticks did not show up in the above suggestion.

Reply:
The above suggestion looks like so:

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

Or

Example to reproduce issue with
``` text
example here
```
_Please make sure to include the rule(s) that cause the issue if applicable_